roomPanel/roomPanel.js

```diff
@@ -130,9 +130,14 @@ CandyShop.RoomPanel = (function(self, Candy, Strophe, $) {
                     Candy.Core.getConnection().sendIQ(iq, self.updateRoomList);
                 } else {
 
+                    var listMaxHeight = '';
+                    if (_options.roomListMaxHeight) {
+                        listMaxHeight = ' style="max-height: ' + _options.roomListMaxHeight + 'px"';
```

Comment:
Hey Ben. Quite the speedy reply!

I initially had this in the template, but it wasn't immediately apparent to me what the best way of getting to the options/getting that information into the template function would be. It also doesn't help that I'm not super familiar with Mustache.

Can you shed some light on the best method to accomplish this?